pikaur/build.py

```diff
@@ -573,6 +573,9 @@ def _install_repo_deps(self) -> None:
         if not self.all_deps_to_install:
             return
 
+        if PikaurConfig().build.SkipRedundantDepRemovals.get_bool():
+            return
```

Comment:
just return-ing from this function doesn't achieving that goal that you intend, also it's missing documentation in the readme file